editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/index.tsx

```diff
@@ -1,40 +1,87 @@
+import { gql, useQuery } from "@apollo/client";
 import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
 import Typography from "@mui/material/Typography";
-import React from "react";
+import React, { useMemo } from "react";
 import EditorRow from "ui/editor/EditorRow";
 
 import { useStore } from "../../../lib/store";
-import { useSubmittedApplications } from "./submissionData";
-import SubmissionsView from "./SubmissionsView";
+import EventsLog from "./EventsLog";
+
+export interface Submission {
+  sessionId: string;
+  eventId: string;
+  eventType:
+    | "Pay"
+    | "Submit to BOPS"
+    | "Submit to Uniform"
+    | "Send to email"
+    | "Upload to AWS S3";
```

Comment:
nit: I feel like the tense of these isn't quite right - should they all be past tense? What's a word that works for _either_ a successful or failed payment event (eg why I didn't go with "Paid" to begin with!)?

Open to suggestions!

Reply:
Yeah this is tricky right! I wonder if "Submission to..." and "Payment" are any better?

"Paid" seems to imply success, but a "Payment" could be successful or failed? See comment in review about success and failure states which relate to this also.

Reply:
No one else mentioned this, so keeping as-is for now :+1: